src/store/reducers/describe.ts

```diff
@@ -42,6 +45,18 @@ const describe = function z(
     }
 };
 
+// Consumers selectors
+const selectConsumersNames = (state: any, path: string): string[] =>
+    state.describe.data[path]?.PathDescription?.PersQueueGroup?.PQTabletConfig?.ReadRules || [];
```

Comment:
same as in the other PR, the base selectors should be reference stable, a new array will cause each dependent selector to recalculate each time